DESCRIPTION

```diff
@@ -9,7 +9,7 @@ Authors@R: c(
            person("Ruixue", "Wang", email = "ruixue.wang@merck.com", role = c("ctb")),
```

Comment:
update my role to "aut", "cre"; remove Keaven's "cre" role. The current way will result in a note. Please refer gsDesign2.